### PR TITLE
Fix alignment and changed bar height

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
Fix alignment and changed bar height

#### What problem is this solving?
Move visual appealing interface

#### How should this be manually tested?
[workspace](https://felipe--storecomponents.myvtex.com)

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react'
+import React, { Component, Fragment } from 'react'
```
Comment (brunojdo) [inline react/index.js]: The `Fragment` isn't used.

Comment (salesfelipe) [reply react/index.js]: 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Bar height
+### Fixed
+- Icon and text alignment,
```
[the remark this answers is not in the document]
Comment (salesfelipe) [reply CHANGELOG.md]: 👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Bar height
```
Comment (brunojdo) [inline CHANGELOG.md]: Make the changelog texts more meaningful

**Pull request merged**